libs/expo/betterangels/src/lib/ui-components/ConsentModal.tsx

```diff
@@ -188,44 +189,24 @@ export default function ConsentModal(props: IMainModalProps) {
             }}
           />
           <Image
+            style={{
+              marginTop: '10%',
```

Comment:
we should not differ position of the image based on the device from top unless it is required.